xclim/testing/tests/test_indices.py

```diff
@@ -1045,6 +1045,41 @@ def test_below_simple(self, tasmax_series):
         np.testing.assert_array_equal(out[1:], [0])
 
 
+class TestJetStreamIndices:
+    # data needs to consist of at least 61 days for Lanczos filter (here: 66 days)
+    time_coords = pd.date_range("2000-01-01", "2000-03-06", freq="D")
+    # make random ua data array of shape (66 days, 3 plevs, 5 lons, 5 lats)
+    np.random.seed(42)
+    da_ua = xr.DataArray(
+        np.random.rand(66, 3, 5, 5),
+        coords={
+            "time": time_coords,
+            "plev": [75000, 85000, 100000],
+            "lon": [-60, -59, -58, -57, -56],
+            "lat": [15, 16, 17, 18, 19],
```

Comment:
I suggest we make sure it works with arbitrary variable names, as long as cf-xarray can recognize the variables (for example based on the standard_name).